icgcget/clients/commands/command_utils.py

```diff
@@ -0,0 +1,34 @@
+import click
+from ..errors import ApiError
```

Comment:
Why are you using relative imports in the first place? Seems to be an anti-pattern:

http://stackoverflow.com/questions/4209641/absolute-vs-explicit-relative-import-of-python-module